appstudio-controller/controllers/appstudio.redhat.com/devsandbox_deployment_controller.go

```diff
@@ -177,6 +179,11 @@ func newDeploymentTarget(deploymentTargetClassName applicationv1alpha1.Deploymen
 		},
 	}
 
+	// Set AllowInsecureSkipTLSVerify if it is a dev cluster.
```

Comment:
I think comment should be --> `// Set AllowInsecureSkipTLSVerify field of DT to True ,if it is a dev cluster.`